platform/mellanox/mlnx-platform-api.mk

```diff
@@ -0,0 +1,7 @@
+# SONIC_PLATFORM_API_PY2 package
+
+SONIC_PLATFORM_API_PY2 = mlnx_platform_api-1.0-py2-none-any.whl
```

Comment:
So the thought here is that each platform vendor can name their Python wheel anything they like as long as it installs a package called "sonic_platform", correct?